skimage/measure/_regionprops.py

```diff
@@ -209,9 +209,16 @@ def __init__(self, slice, label, label_image, intensity_image,
                  cache_active, *, extra_properties=None):
 
         if intensity_image is not None:
-            if not intensity_image.shape == label_image.shape:
-                raise ValueError('Label and intensity image must have the'
-                                 ' same shape.')
+            ndim = label_image.ndim
+            if not (
+                    intensity_image.shape[:ndim] == label_image.shape
+                    and intensity_image.ndim in [ndim, ndim + 1]
+                    ):
```

Comment:
```suggestion
            ):
```
Was it your editor's indentation choice? :thinking: It's *not* PEP8-compliant... :confounded:

Reply:
@mkcor I think it is?

https://www.python.org/dev/peps/pep-0008/#indentation

> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list [...] or it may be lined up under the first character of the line that starts the multiline construct

Reply:
Oh, I wasn't aware of this convention and trusted **pep8speaks**:

* In the file skimage/measure/_regionprops.py:

    Line 216:21: E123 closing bracket does not match indentation of opening bracket's line

Reply:
Yeah sadly sometimes automated tools don't admit the full subtlety of human language!